shell/list/provisioning.cattle.io.cluster.vue

```diff
@@ -88,6 +88,11 @@ export default {
   },
 
   computed: {
+    groupable() {
```

Comment:
this is going to break with Cody's SSP PR. We won't have access to all provisioning clusters (filteredRows) just a single pages worth.